docs/install/assets/eventing.md

```diff
@@ -0,0 +1,11 @@
+# Knative Serving installation files (0.21 releases)
```

Comment:
Remove the specific version `(0.21 releases)` this will be merged into master, and then it can be cherry pick

Reply:
I'll remove the version.